feat: Postgis support in ParadeDB (#1636)
paradedb/paradedb: docs/deploy/extensions.mdx

  This command assumes that your ParadeDB container name is `paradedb`.
</Note>

> Next, install the [prebuilt binaries](https://debian.pkgs.org/sid/debian-main-amd64/postgresql-16-partman_5.1.0-1_amd64.deb.html).

Reviewer: ```suggestion
Next, install the [prebuilt binaries](https://pkgs.org/search/?q=partman).
```

### Add to `shared_preload_libraries`

<Accordion title = "Modifying shared_preload_libraries">

If you are installing an extension which requires this step, you can do so
via the following command, replacing `<extension_name>` with your extension's name:

```bash
sed -i "/^shared_preload_libraries/s/'\([^']*\)'/'\1,<extension_name>'/" /var/lib/postgresql/data/postgresql.conf
```

> So in case of `pg_partman` the command will be:

Reviewer: ```suggestion
For `pg_partman`, the command is:
```

Connect to ParadeDB via `psql` and create the extension.

```sql
CREATE EXTENSION pg_partman;
```

pg_partman is now ready to use!

> Note that this is a very simple example of installing pg_partman, there are more steps you probably want to do listed in the official [install instructions](https://github.com/pgpartman/pg_partman?tab=readme-ov-file#installation)

Reviewer: ```suggestion
Note that this is a simple example of installing `pg_partman`. The full list of settings and optional dependencies can be found in the [official installation instructions](https://github.com/pgpartman/pg_partman?tab=readme-ov-file#installation).
```
